PennMobile/General/Networking + Analytics/UserDBManager.swift

```diff
@@ -110,7 +110,7 @@ extension UserDBManager {
 
 // MARK: - Laundry
 extension UserDBManager {
-    func saveLaundryPreferences(for rooms: [LaundryRoom]) {
+    func saveLaundryPreferences(for rooms: [OldLaundryRoom]) {
```

Comment:
Not a blocker but we should determine if we still have a use for these and delete these if needed.

Also, we should consider adding some sort of migration mechanism if the laundry favorites are now stored and retrieved differently.